Changelog.md

```diff
@@ -1,5 +1,7 @@
 Unreleased
 -------------------------
+* [Bug fix] Remove randomness on how the XBlock attributes are
```

Comment:
"Remove randomness" sounds a little awkward. :) I'd put "Make XBlock exports (from Studio or its REST API) deterministic and predictable."

Reply:
Ack, done.